bfsos/bfsos_test.go

```diff
@@ -0,0 +1,84 @@
+package bfsos_test
+
+import (
+	"fmt"
+	"io/ioutil"
+	"os"
+	"path/filepath"
+	"testing"
+
+	"github.com/bsm/bfs/bfsos"
+	"github.com/bsm/bfs/testdata/lint"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+)
+
+const (
```

Comment:
pls remove the two, I don't like tests to write files to local dirs, always create a brand new base dir via `ioutil.TempFile("", "bfsfs")`